charts/bpa/README.md

```diff
@@ -243,6 +243,7 @@ Note: Deleting the PVC's will delete postgresql data as well. Please be cautious
 | bpa.config.security.enabled | bool | `true` |  |
 | bpa.config.titleOverride | string | `""` | Override title shown in the browser tab. Default: Helm release name, capitalized (or NameOverride if given) |
 | bpa.config.web.only | bool | `false` |  |
+| bpa.config.webhook.key | string | `"changeme"` |  |
```

Comment:
add some small docu to the values file ( format see other parameters) --> will be automatically included here with the helm docu build